lib/logflare/source/bigquery/buffer_counter.ex

```diff
@@ -177,24 +177,26 @@ defmodule Logflare.Source.BigQuery.BufferCounter do
   end
 
   defp broadcast_buffer(state) do
```

Comment:
can shift the Task firing upwards to the callback so that the private function can be the Task arg instead.